pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -214,6 +214,26 @@ func (c *Cluster) startMons(targetCount int) error {
 		}
 	}
 
+	// Enable Ceph messenger 2 protocol on Nautilus
+	if c.clusterInfo.CephVersion.IsAtLeastNautilus() {
+		v, err := client.GetCephMonVersion(c.context)
+		if err != nil {
+			return fmt.Errorf("failed to get ceph mon version. %+v", err)
+		}
+		if v.IsAtLeastNautilus() {
+			versions, err := client.GetCephVersions(c.context)
```

Comment:
Since this is after the mons have been updated, we would expect them all to be updated at this point, right? In what scenario would the mons not all be on nautilus? Is this when the daemon takes time to start since we don't wait for full quorum before continuing?

Reply:
Currently, I don't think this problem exists since we always wait at some point for all the mons to be in quorum.